packages/efl/efl.1.19.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Alexis Bernadet <alexis.bernadet at noos.fr>"
+authors: "Alexis Bernadet <alexis.bernadet at noos.fr>"
+available: [ ocaml-version >= "3.12"]
+homepage: "https://forge.ocamlcore.org/projects/ocaml-efl/"
+license: "LGPL with linking exception"
+dev-repo: "https://github.com/axiles/ocaml-efl.git"
+bug-reports: "https://github.com/axiles/ocaml-efl/issues"
+build: [
+  ["./configure" "--prefix=%{prefix}%" "OCAMLFIND_DESTDIR=%{lib}%"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "efl"]
+depends: [
+  "ocamlfind"
+  "ocamlbuild" {build}
+]
+depexts: [
+  [["osx" "homebrew"] ["efl"]]
+  [["source" "linux"] ["https://gist.githubusercontent.com/axiles/9f586339249397c08a3c3b8b66793a51/raw/52685ac061c36c15ba48f8c9d3fb876fb2bc7c15/install_efl_1_19_on_ubuntu"]]
```

Comment:
Since the [script](https://gist.githubusercontent.com/axiles/9f586339249397c08a3c3b8b66793a51/raw/52685ac061c36c15ba48f8c9d3fb876fb2bc7c15/install_efl_1_19_on_ubuntu) just calls the package manager directly:

```
sudo apt update
sudo apt --yes install libelementary-dev
```

it'd be better to list the `depexts` directly in the `opam` file, like this:
```
  [ ["debian"] [ "libelementary-dev"] ]   
  [ ["ubuntu"] [ "libelementary-dev"] ]
```